xikolo-ios/QuizWebViewController.swift

```diff
@@ -25,12 +25,17 @@ class QuizWebViewController: UIViewController, UIWebViewDelegate {
         activityIndicator.center = CGPoint(x: self.view.frame.size.width / 2, y: self.view.frame.size.height / 3)
         activityIndicator.tag = 100
         
-        if self.url != nil {
-            let url = NSURL(string: self.url!)
+        if let courseID = courseItem.section?.course?.id {
+            let courseURL = Routes.BASE_URL + Routes.COURSES + courseID
+            let quizpathURL = "/items/" + courseItem.id!
+            let wholeURL = courseURL + quizpathURL
+            let url = NSURL(string: wholeURL)
             let request = NSMutableURLRequest(URL: url!)
             request.addValue(Routes.HEADER_USER_PLATFORM_VALUE, forHTTPHeaderField: Routes.HEADER_USER_PLATFORM)
+            request.addValue(Routes.HEADER_AUTHORIZATION_VALUE_SCHEMA + UserProfileHelper.getToken(), forHTTPHeaderField: Routes.HEADER_AUTHORIZATION)
```

Comment:
Please move the request creation somewhere generic and reuse it for the news view (and any others that might still come).